[!] Program not found: seacrh
[!] Program not found: seacrh
[!] Program not found: 10.129.220.0/22
[!] Program not found: [show
[!] Program not found: seacrh
Traceback (most recent call last):
  File "/home/donald/.SuperSploit/exploits/android_tools/adb Interface.py", line 249, in sys_call_Linux
    subprocess.run(dataList)
  File "/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1901, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'exploit'
Traceback (most recent call last):
  File "/home/donald/.SuperSploit/exploits/android_tools/adb Interface.py", line 249, in sys_call_Linux
    subprocess.run(dataList)
  File "/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1901, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: ''
[!] Program not found: return
[!] Program not found: 6
[!] Program not found: exploit\
Traceback (most recent call last):
  File "/home/donald/.SuperSploit/exploits/android_tools/adb Interface.py", line 249, in sys_call_Linux
    subprocess.run(dataList)
  File "/usr/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.11/subprocess.py", line 1901, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'recored_screen'
